### DISCUSSION TITLE
Get-DbaRunningJob

### DISCUSSION THREAD

**Discussion by krishna2722** [dataplat/dbatools#8681] [Q&A] [answered]
The Starttime of the job is in nvarchar datetype, how to convert it while using select-object. Firsty why it has nvarchar datatype for a datetime column. Can someone fix this or provide me a solution. Thanks in advance.

**Comment by wsmelton** [comment, marked as answer]
Please update to the latest release. We use a custom type in our output properties if DbaDateTime and you should be getting that for the command output.
